packages/react/lib/runtime.js

```diff
@@ -4,21 +4,33 @@
 const { createElement, Component } = require('react');
 const { default: withRouter } = require('react-router-dom/es/withRouter');
 
-exports.Miss = withRouter(({ staticContext }) => {
+const { bootstrap } = require('@untool/core');
+
+exports.render = function render(element, options) {
+  return function render(...args) {
+    const { render } = bootstrap({}, element, options);
+    if (!render) {
+      throw new Error("Can't use @untool/react mixin");
```

Comment:
Shouldn't this be `Can't find @untool/react mixin` instead?
Maybe even: `Can't find @untool/react mixin - make sure you have installed @untool/defaults or hops-preset-defaults` or something similar with a helpful message?